default.nix

```
@@ -0,0 +1,16 @@
+# This file provides backward compatibility to nix < 2.4 clients
+{ system ? builtins.currentSystem }:
+let
+  lock = builtins.fromJSON (builtins.readFile ./flake.lock);
+
+  inherit (lock.nodes.flake-compat.locked) owner repo rev narHash;
```

```suggestion
  inherit (lock.nodes.${lock.nodes.root.inputs.flake-compat}.locked) owner repo rev narHash;
```

Strictly speaking this is more correct and safer if more inputs are added; see https://github.com/edolstra/flake-compat/pull/61 for context